docs/versions/oscar-2301.md

```diff
@@ -0,0 +1,287 @@
+# OSCAR 23.01
+
+OSCAR 23.01 is the January 2023 version of the OSCAR Corpus. While being quite similar to OSCAR 22.01, it contains several new features, including [KenLM](https://kheafield.com/code/kenlm/)-based adult content detection, precomputed [Locality-Sensitive Hashes](https://fr.wikipedia.org/wiki/Locality_sensitive_hashing) for near deduplication, and [blocklist](https://dsi.ut-capitole.fr/blacklists/index_en.php)-based categories.
+
```

Comment:
Should we also mention:
1. That we have now moved to `zstd` and give a little info on how to deal with this?
2. That we have an official downloding tool that people can use? (Maybe we should add this to the mail template)
3. Link the to the official toolkit `OSCAR-tools`?